src/marketplace/eqp/v1/packages.md

```diff
@@ -148,17 +148,21 @@ Listing the JSON structure of objects described above:
 
 #### version_compatibility
 
-Assuming a Magento 2 package:
+For a new Magento 2 package:
 
 ```json
 "version_compatibility" : [
   {
     "edition" : "CE",
-    "versions" : ["2.0", "2.1", "2.2"]
+    "versions" : ["2.3", "2.4"]
   },
   {
     "edition" : "EE",
-    "versions" : ["2.0","2.1", "2.2"]
+    "versions" : ["2.2","2.3", "2.4"]
```

Comment:
Since this is for a new M2 package, is "2.2"  a valid version? Same for line 165.

Reply:
Updated.